File: beecrowd/1015/1015 - Distance Between Two Points .c
#include <stdio.h>
#include <math.h>

int main() {
    double x1, y1, x2, y2;
    scanf("%lf %lf", &x1, &y1);
    scanf("%lf %lf", &x2, &y2);

    double dx = x2 - x1;
    double dy = y2 - y1;
    double distance = sqrt(dx*dx + dy*dy);

    printf("%.4lf\n", distance);
    
    return 0;
}

File: beecrowd/1015/1015 - Distance Between Two Points .cs
using System;

class Program
{
	static void Main(string[] args)
	{
		double x1, y1, x2, y2;
		string[] input;
		
		input = Console.ReadLine().Split(' ');
		x1 = double.Parse(input[0]);
		y1 = double.Parse(input[1]);
		
		input = Console.ReadLine().Split(' ');
		x2 = double.Parse(input[0]);
		y2 = double.Parse(input[1]);
		
		double distance = Math.Sqrt(Math.Pow((x2 - x1), 2) + Math.Pow((y2 - y1), 2));
		
		Console.WriteLine($"{distance:F4}");
	}
}